feature-testing/test/features/resources/all-ui.feature

```diff
@@ -0,0 +1,64 @@
+Feature: Work-in-progress dumping ground of DDEV local ui features
+
+   This feature file is home to all ddev local ui features that have not yet had the Given/when/then steps fully caputred. Once steps have been fully captured, we will pull the scenarios out of this feature file and put them in a more descriptive one
+
+Scenario: DDEV-UI requires and checks for an appropriate version of the command line
+Given I have DDEV UI installed
+And I have a compatible version of the command line installed
+When I open up DDEV UI
+Then the command line version check passes
+
+Given I have DDEV UI installed
+And I have an incompatible version of the command line installed
+When I open up DDEV UI
+Then the command line version check fails
+
+Scenario: DDEV-UI requires and checks that Docker is running
+Given I have DDEV UI installed
+And Docker is running
+When I open up DDEV UI
+Then the command line Docker check passes
```

Comment:
So it' not a command line check for docker. It uses the `ddev list` command which under the hood has checks for Docker. So it's more you will pass them initial application system checks and land inside the app when docker is installed and running, if not it will tell you that there was a problem connecting to Docker.

Reply:
Ahh gotcha! This scenario and the other one you commented on in the same feature file havent been filled out yet (theyre pulled right from kory's doc) but ill pop these notes in there right now!